docs/installation.md

```diff
@@ -299,7 +299,7 @@ configuration.
 By default, the Trino Gateway process is started with the following command:
 
 ```shell
-java -XX:MinRAMPercentage=80.0 -XX:MaxRAMPercentage=80.0 -jar /usr/lib/trino/gateway-ha-jar-with-dependencies.jar config.yaml
+java -XX:MinRAMPercentage=80.0 -XX:MaxRAMPercentage=80.0 -jar /usr/lib/trino-gateway/gateway-ha-jar-with-dependencies.jar config.yaml
```

Comment:
Separate this into another commit since its unrelated and check if we refer to that folder elsewhere .. you could also drop this change and move it to another PR

Reply:
Dropped